compiler/src/dotty/tools/dotc/transform/init/Objects.scala

```diff
@@ -821,6 +832,7 @@ class Objects(using Context @constructorOnly):
               errorReadOtherStaticObject(State.currentObject, addr)
               Bottom
           else if ref.isObjectRef && ref.klass.hasSource then
+            println(s"Uninitialized field Position 2, ref = $ref, target = $target")
```

Comment:
You probably didn't intend to leave these printlns in.

Reply:
It has been corrected